Gruntfile.js

```diff
@@ -29,7 +29,8 @@ module.exports = function(grunt) {
     "concat",
     "uglify",
     "postcss",
-    "cssmin"
+    "cssmin",
+    "purifycss"
   ]);
```

Comment:
Can we also add it to the deploy task please?

Reply:
Right now we are generating a CSS which is stripped, which we also can use as our `*.min.css` version, but I am not fully trusting that one yet. 

The purify task is showing the not used classes right now.